src/v0/parity.rs

```diff
@@ -239,32 +240,32 @@ mod test {
 
     #[test]
     fn it_is_a_valid_usage() {
-        let holder = [1u8; 32];
+        let holder = [1_u8; 32];
         let doughnut = DoughnutV0 {
-            issuer: [0u8; 32],
+            issuer: [0_u8; 32],
             holder,
-            domains: Default::default(),
+            domains: Vec::default(),
             expiry: make_unix_timestamp(10),
             not_before: 0,
             payload_version: 0,
             signature_version: 0,
-            signature: Default::default(), // No need to check signature here
+            signature: primitive_types::H512::default(), // No need to check signature here
```

Comment:
Add the `use primitive_types::H512;` to the top of the file and change to `signature: H512::default()`
Same with the other usages